wdn/templates_5.0/examples/audioplayer.html

```diff
@@ -1,17 +1,17 @@
 <!DOCTYPE html>
 <html>
-<head>
+  <head>
     <meta http-equiv="content-type" content="text/html;charset=UTF-8" />
     <title>Audio Player Example</title>
-</head>
-<body data-version="4.1">
+  </head>
+  <body data-version="4.1">
```

Comment:
This should be `data-version="5.0"`.  Need to update all occurrences of in other files also.

Reply:
I was waiting until the corresponding documentation was updated to change the `data-version` for each example but, in retrospect, I'm not sure it makes much of a difference. I'll update the rest.